src/CaptureClient/ApiEventProcessor.cpp

```diff
@@ -27,132 +27,192 @@ inline std::string DecodeString(const Source& encoded_source) {
                                  encoded_source.encoded_name_additional_size());
 }
 
-inline TimerInfo TimerInfoFromEncodedEvent(const orbit_api::EncodedEvent& encoded_event,
-                                           uint64_t start, uint64_t end, int32_t pid, int32_t tid,
-                                           uint32_t depth) {
-  TimerInfo timer_info;
-  timer_info.set_start(start);
-  timer_info.set_end(end);
-  timer_info.set_process_id(pid);
-  timer_info.set_thread_id(tid);
-  timer_info.set_depth(depth);
-  timer_info.set_type(TimerInfo::kApiEvent);
-  timer_info.mutable_registers()->Reserve(6);
-  timer_info.add_registers(encoded_event.args[0]);
-  timer_info.add_registers(encoded_event.args[1]);
-  timer_info.add_registers(encoded_event.args[2]);
-  timer_info.add_registers(encoded_event.args[3]);
-  timer_info.add_registers(encoded_event.args[4]);
-  timer_info.add_registers(encoded_event.args[5]);
-  return timer_info;
+void EncodedColorToColor(uint32_t encoded_color, orbit_client_protos::Color* color) {
+  color->set_red((encoded_color & (0xff << 24)) >> 24);
```

Comment:
I think this is the reverse order, due to little-endianness, RGBA is encoded ABGR in memory. As this is the same code as in EncodedColorToColor, this would explain why I'm seeing flipped colors with manually instrumented scopes from earlier version of the Orbit api.